src/guides/v2.3/coding-standards/code-standard-less.md

```diff
@@ -740,6 +740,86 @@ For mixins grouping use the double underscore "__" prefix.
 }
 ```
 
+Here are common situations when different elements use a similar set of CSS properties
+
+In a .css file, it was necessary to copy such properties for each element. In a .less file, it can be done by reusing the CSS rule – using mixins.
+
+For example, Many elements on the page will have similar animation. For this, you can create an .animation-1 class with a set of animation properties:
+
+```css
+.animation-1 {
+    transition: 300ms ease-in-out;
+    -moz-transition: 300ms ease-in-out;
+    -webkit-transition: 300ms ease-in-out;
+    -o-transition: 300ms ease-in-out;
+}
+```
+
+And then apply this mixin in places where it is necessary:
+
+```css
+.example-1 {
+   .animation-1();
+   width: 100%;
+}
+```
+
+As a result, after compiling the .less file into a .css file, the .example-1 element will have the following:
+
+```css
+.animation-1 {
+   transition: 300ms ease-in-out;
+   -moz-transition: 300ms ease-in-out;
+   -webkit-transition: 300ms ease-in-out;
+   -o-transition: 300ms ease-in-out;
+}
+.example-1 {
+   transition: 300ms ease-in-out;
+   -moz-transition: 300ms ease-in-out;
+   -webkit-transition: 300ms ease-in-out;
+   -o-transition: 300ms ease-in-out;
+   width: 100%;
+}
+```
+
+### Mixins with parameters
+
+There are also mixins with parameters. Calling these mixins, we pass them parameter values. Creating such mixin, we recommend setting parameter values by default (since there might occur problems when calling the mixin without specifying the parameter value). Let’s make a mixin with parameters from the example above and see how to call it:
```

Comment:
```suggestion
There are also mixins with parameters. When calling these mixins, we pass them parameter values. When creating this type of mixin, we recommend setting parameter values by default (since problems might occur when calling the mixin without specifying the parameter value). The example below shows how to create a mixin with parameters based on the example above and shows how to call it:
```